tests/screenshot/apps/components/Button.js

```diff
@@ -11,7 +11,9 @@ const ButtonTests = [
 		textSize: 'large',
 		component: <Button>click me</Button>
 	},
+	// [QWTC-2232] Checking of the expanded button will be confirmed in other test cases(QWTC-1832).
```

Comment:
Do you mean the effect of the button when it is clicked? I think it is not related to the QWTC-2232.

Reply:
This comment displays information related to duplicate steps.